test/test_mimic_explainer.py

```diff
@@ -295,6 +295,15 @@ def test_dataframe_model(self, mimic_explainer, sample_cnt_per_grain, grains_dic
             kwargs['categorical_features'] = ['fruit']
         mimic_explainer(model, X, LGBMExplainableModel, features=features, model_task=model_task, **kwargs)
 
+    @pytest.mark.parametrize('if_predictions_as_dataframe', [True, False])
+    def test_explain_model_binary_classification_with_different_format_predictions(
```

Comment:
please add test for multiclass surrogate model as well, as the code there may break on this output type as well.
Probably a good idea to add a test for regression models too, although it seems like that might be safer - not sure if all surrogate model types support dataframe as labels

Reply:
Done.